codalab/rest/users.py

```diff
@@ -239,3 +239,31 @@ def update_users():
     # Return updated users
     users = local.model.get_users(user_ids=[users[0]['user_id']])['results']
     return AdminUserSchema(many=True).dump(users).data
+
+
+@patch('/user/increment_disk_used', apply=AuthenticatedPlugin(), skip=UserVerifiedPlugin)
+def increment_user_disk_used():
+    """
+    Update the disk used for the user who makes the request to this endpoint.
+
+    This is required because users who are bypassing the server to upload
+    files straight to Azure will need their client to tell the server
+    to increment their disk used as file chunks are uploaded. They cannot
+    use the users/ PATCH endpoint since disk_used is in
+    USER_READ_ONLY_FIELDS. We make this special function (which only allows
+    positive disk increments so that users can't decrement their disk used) to ensure
+    that we can safely increment user disk used without introducing a
+    security flaw.
```

Comment:
Add documentation as to what the request format should look like (i.e., the `disk_used_increment` key)

Reply:
Done.